.travis.yml

```diff
@@ -7,9 +7,12 @@ python:
     - "3.5"
     - "3.6"
 env:
+  matrix:
   - DJANGO_MAX=1.11.100 DB=postgres TEST=pulp
   - DJANGO_MAX=2.2.100 DB=postgres TEST=pulp
   - TEST=docs
+  global:
+    secure: kzbI/DagIGIOFjVEyLyYGEi/xVwiq5M8FCEM3mOtXK9ocYs1Ky1ABkcgLrOLBb+S7SMNjy4XnUWGUxhBGWQLDw2/OimKFoHpCHGt1Cb4rXhWDSVPGgDXQzyW8pJsPKtAbemfx5a4oRP82vmpgB/htJ+8dE5CIAZWU8d9dbvQbro=
```

Comment:
This isn't meant to be a secret or anything correct?

Reply:
This is a secure secret- it's our PYPI password that has been encrypted with the travis pulp/pulp key. https://docs.travis-ci.com/user/environment-variables/#Defining-encrypted-variables-in-.travis.yml